symfony/client/src/orangehrmAdminPlugin/pages/location/EditLocation.vue

```diff
@@ -0,0 +1,273 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6" class="orangehrm-main-title">Edit Location</oxd-text>
+
+      <oxd-divider />
+
+      <oxd-form novalidate="true" :loading="isLoading" @submitValid="onSave">
+        <oxd-grid :cols="1">
+          <div>
+            <oxd-form-row>
+              <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+                <oxd-grid-item>
+                  <oxd-input-field
+                    label="Name"
+                    v-model="location.name"
+                    :rules="rules.name"
+                    required
+                    :disabled="!hasUpdatePermissions"
+                  />
+                </oxd-grid-item>
+                <oxd-grid-item>
+                  <oxd-input-field
+                    type="dropdown"
```

Comment:
Consider using the [new dropdown](https://rajithakumara-oxd.netlify.app/?path=/story/example-selectinput--default&globals=measureEnabled:false). It has the same API, and the v-model will return selected object instead of array.

`<oxd-input-field type="select"  label="Country" :options="countries" v-model="location.countryCode" :rules="rules.countryCode" :disabled="!hasUpdatePermissions" required />`

Reply:
Done.